Digipost.Signature.Api.Client.Direct/DirectClient.cs

```diff
@@ -78,6 +81,47 @@ private static Uri RelativeUrl(Job job)
             }
         }
 
+        public async Task<JobStatusResponse> GetStatusChange(Sender sender = null)
```

Comment:
Missing documentation here.  

I see that almost all of this is an exact copy of the portal code. In this case, maybe this is okay since they may diverge in the future.